code/datums/diseases/advance/symptoms/cough.dm

```diff
@@ -32,7 +32,7 @@ BONUS
 			if(1, 2, 3)
 				to_chat(M, "<span notice='notice'>[pick("You swallow excess mucus.", "You lightly cough.")]</span>")
 			else
-				M.emote("cough")
+				M.emote("cough", , , 1)
```

Comment:
If you do double check all the overrides of the proc because I believe we've had a runtime related to that in the past.